File: Bronze/2720.cpp
#include <bits/stdc++.h>

using namespace std;



int main(){
    ios::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int quarter = 25, dime = 10, nickel = 5, penny = 1;
    int t;
    cin >> t;

    while(t--) {
        int cash;
        int q, d, n, p;

        cin >> cash;
        
        q = cash / quarter;
        cash %= quarter;

        d = cash / dime;
        cash %= dime;

        n = cash / nickel;
        cash %= nickel;

        p = cash / penny;

        cout << q << " " << d << " " << n << " " << p << "\n";
    }

    return 0;
} 

File: Bronze/2720.py
import sys

input = sys.stdin.readline

quarter = 25
dime = 10
nickel = 5
penny = 1

for _ in range(int(input())):
  c = int(input())
  q = c // quarter
  d = c % quarter // dime
  n = c % quarter % dime // nickel
  p = c % quarter % dime % nickel // penny
  print(q, d, n, p)
